core/src/main/java/io/grpc/internal/ManagedChannelImpl.java

```diff
@@ -187,7 +187,12 @@ public void uncaughtException(Thread t, Throwable e) {
               Level.SEVERE,
               "[" + getLogId() + "] Uncaught exception in the SynchronizationContext. Panic!",
               e);
-          panic(e);
+          try {
+            panic(e);
+          } catch (Throwable t) {
+            logger.log(
+                Level.SEVERE, "[" + getLogId() + "] Uncaught exception while panic()ing", e);
```

Comment:
Did you intend to have the parenthesis instead of a 'k'?

Reply:
Yeah, panic() as the function call. Seems the same either way.